conf.d/health.d/load.conf

```diff
@@ -0,0 +1,53 @@
+
+# you can disable an alarm notification by setting the 'to' line to: silent
+
+# Calculate the base trigger point for the load average alarms.
+# This is the maximum number of CPU's in the system over the past 1
+# minute, with a special case for a single CPU of setting the trigger at 2.
+template: load_trigger
+      on: system.cpu
+      os: linux freebsd
```

Comment:
nice!

All these are linux specific unfortunately. I haven't added the $processors variable to freebsd (I don't know how to get the active processors). I will check that with @vlvkobal .

Reply:
Ah, OK, I hadn't realized that. 

I'm pretty sure you can query the sysctl value `hw.ncpu` to get the number of logical CPU's on a FreeBSD system, but I don't know if that tracks _active_ CPU's (which we need here), or _total present_ CPU's.

Reply:
Correction, the required sysctl value on FreeBSD is `kern.smp.cpus`.  `hw.ncpus` tracks physically present logical cores, while that tracks actually used logical cores.

Reply:
ok. until this is done, please remove freebsd.
You also need to have `os` lines to all these alarms, as they depend on the same thing.

Reply:
Huh, though I had put `os` lines on all of them.  Guess not.